src/psyclone/psyir/nodes/call.py

```diff
@@ -47,11 +47,27 @@
 from psyclone.psyir.nodes.reference import Reference
 from psyclone.psyir.nodes.routine import Routine
 from psyclone.psyir.symbols import (
-    RoutineSymbol, Symbol, SymbolError, UnsupportedFortranType)
+    RoutineSymbol,
+    Symbol,
+    SymbolError,
+    UnsupportedFortranType,
+    DataSymbol,
+)
+from typing import List
+from psyclone.errors import PSycloneError
+
+
+class CallMatchingArgumentsNotFound(PSycloneError):
+    """Exception to signal that matching arguments have not been found
+    for this routine
+    """
+    def __init__(self, value):
+        PSycloneError.__init__(self, value)
+        self.value = "CallMatchingArgumentsNotFound: " + str(value)
 
 
 class Call(Statement, DataNode):
-    ''' Node representing a Call. This can be found as a standalone statement
+    """Node representing a Call. This can be found as a standalone statement
```

Comment:
Please could you revert these formatting changes (the change from ''' to """). The smaller a PR is, the fewer conflicts we get.

Reply:
OK

Reply:
It would be nice to have an agreement on what to you. It's again and again different across PSyclone.

Reply:
Oh. Sorry, I see it's all over this file.
I assume it was done automatically by some reformatting tool. I'll undo it.